opal/runtime/opal.h

```diff
@@ -111,6 +114,19 @@ OPAL_DECLSPEC void opal_warn_fork(void);
  */
 OPAL_DECLSPEC int opal_register_params(void);
 
+/**
+ * Wrapper to return the hostname value that is in
+   opal_process_info.nodename, as opposed to calling gethostname()
+   directly, which is not guaranteed to be null-terminated and
+   varies in its behavior depending on implementation. The 
+   opal_process_info.nodename value is first populated in 
+   opal/runtime/opal_init.c
+ */
+OPAL_DECLSPEC static inline const char *opal_gethostname( void ) {
```

Comment:
```suggestion
static inline const char *opal_gethostname( void ) {
```

Cannot have the `OPAL_DECLSPEC` there -- that's only for non-static symbols (for me, it results in a compiler warning every time the file is included).

Reply:
I'm sorry, I think my final review comment was misleading / I didn't reference this minor issue.  We can't have this OPAL_DECLSPEC here.  I took the liberty of amending the commit to fix this issue.  We should be good to go now.